src/screens/Profile/Header.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {
+  View,
+  StyleSheet,
+} from 'react-native';
+import {
+  Title, Button,
+} from 'react-native-paper';
+import PropTypes from 'prop-types';
+import { withNavigation } from 'react-navigation';
+
+
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onBackPress = this.onBackPress.bind(this);
+  }
+
+  onBackPress() {
+    const { navigation } = this.props;
+    navigation.navigate('ProfileHome');
```

Comment:
`navigation.goBack()` might work but i'm not sure

Reply:
For me: It did not unfortunately :/

Reply:
Does `navigation.goBack(null)` work? I have not used this function and the API is very confusing, but I would like to try it out: https://reactnavigation.org/docs/4.x/navigation-prop#goback---close-the-active-screen-and-move-back